### PR TITLE
Update the deps and tests.

### PR DESCRIPTION
**Pull request by solarhell**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by solarhell** [comment]
稍等我修复下lint..

File: sequelize/test/app/service/post.test.js [hunk 1]
```diff
@@ -144,7 +144,7 @@ describe('test/app/service/post.test.js', () => {
 
     it('should throw 404 when id not found', async () => {
       try {
-        await ctx.service.post.destroy({ id: 1 });
+        await ctx.service.post.destroy({ id: 1, user_id: 1 });
```
Comment (dead-horse) [inline sequelize/test/app/service/post.test.js]: 这里为什么加了 user_id: 1

Comment (solarhell) [reply sequelize/test/app/service/post.test.js]: 去掉会发现测试跑不通测试，而且这个函数签名确实要求传入user_id

Comment (solarhell) [reply sequelize/test/app/service/post.test.js]: ping @dead-horse

Comment (dead-horse) [reply sequelize/test/app/service/post.test.js]: 测试报什么错？

Comment (solarhell) [reply sequelize/test/app/service/post.test.js]:
```
Error{message:"WHERE parameter \"user_id\" has invalid \"undefined\" value"}
```

Comment (solarhell) [reply sequelize/test/app/service/post.test.js]: 这个还合吗

**Comment by solarhell** [comment]
没人看那就关了

**Pull request closed without merging**